### PR TITLE
CI updates for python3

### PR DESCRIPTION
**Pull request by frozencemetery**
- python-cjson seems dead upstream and in particular has no python3 version: https://github.com/AGProjects/python-cjson/issues/6
- Update paste dependency to python3
- Fix pyrad-depdendent tests

### CODE REVIEW

File: src/tests/jsonwalker.py [hunk 1]
```diff
@@ -1,10 +1,6 @@
 import sys
-try:
-    import cjson
-except ImportError:
-    print("Warning: skipping audit log verification because the cjson module" \
-          " is unavailable")
-    sys.exit(0)
+import json
+
```
Comment (greghudson) [inline src/tests/jsonwalker.py]: We don't normally put a blank line here.

File: src/tests/jsonwalker.py [hunk 2]
```diff
@@ -93,17 +89,15 @@ def _walk(self, adict):
         with open(options.filename, 'r') as f:
             content = list()
             for l in f:
-                content.append(cjson.decode(l.rstrip()))
+                content.append(json.loads(l.rstrip()))
         f.close()
     else:
-        print('Input file in jason format is required')
+        print('Input file in json format is required')
```
Comment (greghudson) [inline src/tests/jsonwalker.py]: Capitalize JSON while we're fixing this.

File: src/tests/jsonwalker.py [hunk 3]
```diff
@@ -93,17 +89,15 @@ def _walk(self, adict):
...
         exit()
 
     defaults = None
     if options.defaults is not None:
-        with open(options.defaults, 'r') as f:
-            defaults = cjson.decode(f.read())
-        f.close()
+        defaults = json.load(open(options.defaults, 'r'))
```
Comment (greghudson) [inline src/tests/jsonwalker.py]: Why remove the ``with ...``?  Leaking the file is harmless but there's no reason to add a harmless leak in a fix like this.  (However, the ``f.close()`` was wrong; I'm not sure f is even in scope at that point.)

Comment (frozencemetery) [reply src/tests/jsonwalker.py]: Whoops, meant to make one change or the other and did both... will fix, thanks!

**Comment by frozencemetery** [comment]
Updated.  The easiest solution I could think of was modifying krad's Makefile to filter output, but that seemed not the right approach.  Open to suggestion.

**Pull request merged**